vcs/template.py

```diff
@@ -1175,49 +1190,12 @@ def drawTicks(self, slab, gm, x, axis, number,
                                 end < -180.:
                             end = wc[0]
                         xs.append([wc[0], end])
-                        tys.append(l_tmp)
-                        txs.append(wc[0])
-                    tstring.append(loc[l_tmp])
-        # now does the mini ticks
-        mintics = getattr(gm, axis + 'mtics' + number)
-        if mintics != '':
```

Comment:
Why is this chunk removed? (if mintics != '')

Reply:
that was the guilty part, it was running the mintics with the same property as the main tics, i now call this thing once for main tics and once for mintics (new input keyword) so we can use the different properties.